pkgs/development/compilers/ghc/9.2.1.nix

```diff
@@ -185,6 +187,9 @@ stdenv.mkDerivation (rec {
     export NIX_LDFLAGS+=" -rpath $out/lib/ghc-${version}"
   '' + lib.optionalString stdenv.isDarwin ''
     export NIX_LDFLAGS+=" -no_dtrace_dof"
+
+    # GHC tries the host xattr /usr/bin/xattr by default which fails since it expects python to be 2.7
+    export XATTR=${lib.getBin xattr}/bin/xattr
```

Comment:
This is incorrect, the `XATTR` env var is not yet supported by GHC's 9.2 branch, see https://gitlab.haskell.org/ghc/ghc/-/merge_requests/6862